docs/marketplace/add_to_marketplace/manage_marketplace_items.md

```diff
@@ -0,0 +1,118 @@
+---
+id: manage_marketplace_items
+title:  Manage Marketplace Items
+sidebar_label:  Manage Marketplace Items
+---
+
+There are three main methods to create, modify and delete Marketplace items:
+
+* (recommended) Use [`miactl`](/docs/cli/miactl/overview), the Mia-Platform command line interface tool.
+* Open an issue on Mia-Platform [Github community page](https://github.com/mia-platform/community).
+* (deprecated) Use the [CMS](/business_suite/guide_cms.md).
+
+## Use `miactl`
```

Comment:
Remember to point out what happens with Windows

Reply:
I would not do it here, I would do it in miactl docs, along with the info of the other OSes. I'll open a PR shortly on miactl repo

Reply:
Here it is mia-platform/miactl#105